index.html

```diff
@@ -167,6 +167,28 @@ <h1 class="about-title">About Myself</h1>
                 </ul>
             </section>
             </section>
+
+            <section class="last-section-mobile">
+                <h2>Contact me</h2>
+                    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+                    <form action="#" method="post">
```

Comment:
Please sign up for the FormSpree service and add it here.  So you can get emails from your contact form. This line needs to look similar to the line below.
```<form action="https://formspree.io/f/{form_id}" method="post">```

Reply:
Thanks for the review @huseyinbiyik